2021/44xxx/GSD-2021-44228.json

```diff
@@ -4439,31 +4520,31 @@
                         },
                         {
                             "name": "https://psirt.global.sonicwall.com/vuln-detail/SNWLID-2021-0032",
-                            "refsource": "CONFIRM",
+                            "type": "EVIDENCE",
                             "tags": [
                                 "Third Party Advisory"
                             ],
                             "url": "https://psirt.global.sonicwall.com/vuln-detail/SNWLID-2021-0032"
                         },
                         {
                             "name": "https://www.oracle.com/security-alerts/alert-cve-2021-44228.html",
-                            "refsource": "CONFIRM",
+                            "type": "EVIDENCE",
                             "tags": [
                                 "Third Party Advisory"
                             ],
                             "url": "https://www.oracle.com/security-alerts/alert-cve-2021-44228.html"
                         },
                         {
                             "name": "FEDORA-2021-f0f501d01f",
-                            "refsource": "FEDORA",
+                            "type": "FEDORA",
```

Comment:
A type of FEDORA, MLIST, MISC, REDHAT, FULLDISC, etc doesn't make much sense.

Update to schema compliant options, and dump any valuable metadata into a different field for later processing?

Reply:
That is part of the cve.org namespace data, we don't touch that.

Reply:
Also the file I submitted passes schema checks, you can verify yourself with the validate-json-file.py

Reply:
If we're keeping it verbatim, then we shouldn't be changing refsource to type: https://github.com/CVEProject/cvelist/blob/master/2021/44xxx/CVE-2021-44228.json#L174